src/submission/models.py

```diff
@@ -1399,6 +1399,11 @@ def author_list(self):
         else:
             return ", ".join([author.full_name() for author in self.authors.all()])
 
+    def bibtex_author_list(self):
+        return " and ".join(
+            [author.full_name() for author in self.frozen_authors()],
+        )
```

Comment:
I believe the spec recommends `"AND"` (all uppercase) as the separator : https://bibtex.eu/fields/author/